crt_portal/cts_forms/templates/forms/complaint_view/show/activity_stream.html

```diff
@@ -19,5 +19,16 @@
         </p>
       </li>
     {% endfor %}
+    <li class="activity-stream-item">
+      <div class="display-flex flex-justify">
+        <b>{% firstof data.author_user.email data.author  %}</b>
```

Comment:
We don't show email in the activity feed, we are showing user name. So I think we either want to add the same logic there or make it username here.

Reply:
Agh good catch I didn't notice that inconsistency when implementing. 

The originating [issue](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/427) calls for email, and I feel like I'm missing some context around if `username` is a helpful identifier as-is or if email would be more helpful. We've got the code here for both approaches 🤷 

Who can make that call? 😄

Reply:
Per Lalitha, we'll continue with `username` across the board. Update incoming...

Reply:
So the backstory on that- In theory we should get the email and name claims from DOJ SSO but it isn't working. We got sign on going and didn't really have more time to dig into it. Since username will always be populated that is what we went with showing. In the future, emails could be added manually or we could reach out to JMD to debug the claims issue.